custom-wp-notify.php

```diff
@@ -11,6 +11,8 @@
  * @package         Custom_Wp_Notify
  */
 
+use BCcampus;
```

Comment:
This gives a warning. Can safely remove as it's not needed. "Warning: The use statement with non-compound name"